wp-content/themes/core/components/card/css/card.pcss

```diff
@@ -157,6 +159,12 @@
 	@mixin t-body-small;
 
 	color: var(--color-grey);
+
+	/* CASE: Dark Theme */
+	.t-theme--dark & {
+		color: var(--color-white);
+		opacity: 0.8;
```

Comment:
@ScottGarrison Why is the whole card's opacity set here? Wouldn't we only want to adjust the background color opacity?

Reply:
Looks like this isn't the whole card, it's just the card date. The original designs have an off grey color for the date instead of straight black. This was a bit of a workaround to keep that styling the same.